portals/knowledge-management/src/Components/Tags.vue

```diff
@@ -44,7 +44,7 @@
 </script>
 
 <template>
-    <div class="flex flex-wrap gap-2">
+    <div class="flex flex-wrap gap-2 text-xs font-medium me-2 py-0.5 rounded">
         <Badge v-for="tag in tags" :key="tag.id" :value="tag.name" />
     </div>
```

Comment:
I think we may actually want to make this change in the `Badge` component right?

This `Tags` component is just to pass an array and make a bunch of tag Badges.

Reply:
Hi,
Done.
Thank You!!